frontend/add-client/add-client.component.tsx

```diff
@@ -142,7 +143,7 @@ export type ClientState = {
   clientSource?: ClientSources | string;
   couldVolunteer?: boolean;
   // Intake services
-  intakeServices?: number[];
+  intakeServices?: IntakeService[];
```

Comment:
In order for the confirm page to show the name of the service, we need to store the full service object, not just the number.